### PR TITLE
Python 3.6 Compatability

### PR DESCRIPTION
**Pull request by brent-stone**
Python 3.6 requires dictionary.items() to be placed into a list in order to iterate over those items. There was also some non-obvious issue using a lambda with map instead of a 'simple' loop for highlighting the set of hexes using the Selection helper class. Finally, the sys.path.append("..") trick wasn't working on my machine. I made the import calls explicit with the assumption that the parent directory is named Hexy. Not ideal... but relative imports weren't working otherwise.

### CODE REVIEW

File: examples/example.py [hunk 1]
```diff
@@ -60,7 +59,7 @@ def __init__(self, size=(600, 600), hex_radius=22, caption="ExampleHexMap"):
         self.hex_apothem = hex_radius * np.sqrt(3) / 2
         self.hex_offset = np.array([self.hex_radius * np.sqrt(3) / 2, self.hex_radius])
 
-        self.hex_map = hx.HexMap()
+        self.hex_map = HexMap()
```
Comment (RedFT) [inline examples/example.py:62]: You've got `import hexy as hx` on line 6, so `hx.HexMap()` shouldn't be broken. I don't see the need for this particular change.

File: examples/example.py [hunk 2]
```diff
@@ -151,7 +150,8 @@ def draw(self):
         # show all hexes
         hexagons = self.hex_map.values()
         hex_positions = np.array([hexagon.get_draw_position() for hexagon in hexagons])
-        sorted_idxs = np.argsort(hex_positions[:,1])
+        sorted_idxs = np.argsort(hex_positions[:, 1])
+        hexagons = list(hexagons)  # need to place dictionary.values() into a list in order to iterate.
```
Comment (RedFT) [inline examples/example.py:154]: Could be placed right after line 151

**Comment by RedFT** [comment]
Hi thanks for working on this! I originally wanted the user be able to easily use the library by simply adding `import hexy as hx` and then using `hx` as the namespace to access everything in this library (like how people normally use numpy).

There should be a way to make hexy python 3.6 compatible without compromising on this, no?

**Comment by breznak** [comment]
Is this PR about especially python3.6 compatibility, or overall python 3 support? If the latter, this PR should be a priority, as python 2 is going to be obsoleted since end of the year.

**Comment by RedFT** [comment]
Python 3 support would be nice indeed 👍  @brent-stone are you still interested in working on this?

**Comment by brent-stone** [comment]
Hi guys, 
This pull request includes the change to get everything working with the latest Python3 distro. I'm not sure exactly what version of Python 3 this code began to break.

Everything else in Hexy appears to work fine if you merge this pull.

Sorry it breaks your import statements. I couldn't figure out a way to keep those =/

**Comment by breznak** [comment]
Since #8 , #9 this PR is now obsoleted (examples removed), on the other hand, it should be moved to hexyexamples repo. 

Btw, KUDOs for Python 3 support for the hexy library!

**Pull request closed without merging**